modular_skyrat/modules/cum/effects/cumface.dm

```diff
@@ -0,0 +1,56 @@
+//you got cum on your face bro *licks it off*
+/datum/component/cumfaced
+	dupe_mode = COMPONENT_DUPE_UNIQUE_PASSARGS
+
+	var/mutable_appearance/cumface
+
+/datum/component/cumfaced/Initialize()
+	if(!is_type_in_typecache(parent, GLOB.creamable))
+		return COMPONENT_INCOMPATIBLE
+
+	SEND_SIGNAL(parent, COMSIG_MOB_CUMFACED)
+
+	cumface = mutable_appearance('modular_skyrat/modules/cum/cumface.dmi')
+
+	if(ishuman(parent))
+		var/mob/living/carbon/human/H = parent
+		if(H.dna.species.limbs_id == "lizard")
+			cumface.icon_state = "cumface_lizard"
```

Comment:
Any way to make this the default for anyone with a snout, similar to how gasmasks render the appropriate sprite for folks that have snoots?

Just considering that this logic means that if you set species to anthro and give yourself, eg. the skulldog snout, it'll use the human cumface sprite.

Reply:
i just went and added that in, thanks for the suggestion

Reply:
Cheers, thanks much!

Reply:
Hey @linnpap  I noticed briefly that even though you have something along the lines of modular facial sprites, wouldn't it also make sense for the different ball size sprites to warrant different sizes for the amounts? 

If possible even, the size play aspects of sprite scaling could also affect the 'size' of it as well as that's not too far fetched.

Reply:
i'll see what i can do

Reply:
ball size now affects the amount of cum you produce, and if more than 10u of it touches you, more of it will be visible on your face